ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/contentassist/BuildPropertiesContentAssistProcessor.java

```diff
@@ -54,4 +59,26 @@ public ICompletionProposal[] computeCompletionProposals(ITextViewer viewer, int
 		}
 		return null;
 	}
+
+	@SuppressWarnings("nls")
+	public Image getImage(String element) {
+		ImageDescriptor desc = null;
+		desc = switch (element)
+			{
+			case "src.includes", "src.excludes", "src.additionalRoots", "permissions", "root.", "root", ".permissions.", ".link", "folder.", ".folder.", "link", "source.", "sourceFileExtensions", "bin.includes", "bin.excludes", "javacCustomEncodings.", "javacDefaultEncoding." -> PDEPluginImages.DESC_CATEGORY_OBJ;
```

Comment:
is this the result of formatting? if so, ignoring this line alone from formatting and manually breaking the line into 2 or 3 looks like a better thing to do?

Reply:
Yes that line is a bit long. I suggest to put comments `//` at the end of the manually broken line to prevent the formatter to merge them again.